man/docker-exec.1.md

```diff
@@ -28,6 +29,9 @@ container is unpaused, and then run
 **-d**, **--detach**=*true*|*false*
    Detached mode: run command in the background. The default is *false*.
 
+**--detach-keys**=""
+  Define the key sequence which detaches the container.
```

Comment:
`Specify a custom key sequence for detaching a container. Format is ctrl-<value> where <value> is one of: a-z, @, ^, [, ``, or _`